### PR TITLE
Add apparmor service check

### PR DESCRIPTION
**Pull request by hjluo**
Add apparmor service check.

- Related ticket: https://progress.opensuse.org/issues/55016
- Needles: None.
- Verification run: service_check:http://10.161.8.44/tests/389 http://10.161.8.44/tests/398
                             security/apparmor:https://openqa.suse.de/tests/3257550
                             verified on sle12sp2 to sle12sp5 http://10.161.8.44/tests/454/

### CODE REVIEW

**Comment by okurz** [comment]
Please remove the dot at the EOL of the commit message subject line

File: lib/services/apparmor.pm [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+use base "apparmortest";
+use testapi;
+use utils;
+use version_utils 'is_sle';
+use warnings;
+use strict;
+
+sub install_service {
+    # apparmor need be installed by pattern
+    zypper_call 'in -t pattern apparmor';
```
Comment (coolgw) [inline lib/services/apparmor.pm]: check with security lily for how to correctly install apparmor in 12sp2/3/4, 15 etc..

Comment (lilyeyes) [reply lib/services/apparmor.pm]: I only can confirm AppAmror is a pattern on Tumbleweed, sle12sp4+ and sle15+. Not sure on other versions/products.

**Comment by lilyeyes** [comment]
The changes look good to me since the code of AppArmor part is moved from original ones and stays unchanged, and also offered verify run.

**Comment by coolgw** [comment]
For validation run, i suggest we need at least run scenario 12sp2->12sp5.
Another advise: for this PR apparmor service check integration task, we only select some simple command check, in future should add test cases with configuration(validate the consistence of configuration file before/after migration).

**Comment by lemon-suse** [comment]
LGTM.

**Pull request merged**